examples/mixed-precision-ir/doc/kind

```diff
@@ -0,0 +1 @@
+basic
```

Comment:
I would maybe put this under a new heading mixed precision as we probably will have more examples doing mixed/adaptive precision. You can also change the adaptive-precision block jacobi to mixed-precision, I think

Reply:
That would mean adding a new style to `doc/scripts/examples.pl`, and adaptive-precision block-jacobi is already somewhat sensibly placed in the `preconditioner` group. How about we change this when we have more mixed precision examples?

Reply:
yeah, sure. I am okay with that.